adapters/Fyber/FyberAdapter/GADMAdapterFyberBannerAd.m

```diff
@@ -124,9 +124,9 @@ - (void)loadBannerAd {
 
   IAAdRequest *request =
       GADMAdapterFyberBuildRequestWithSpotIDAndAdConfiguration(spotID, _adConfiguration);
+    IASDKCore.sharedInstance.mediationType = IAMediationAdMob.new;
```

Comment:
@TomerFyber Can you update to use `alloc] init];` instead?
The same applies to the other files.